packages/terra-i18n/src/intlLoaders.js

```diff
@@ -12,10 +12,19 @@ const supportedIntlConstructors = () => {
   let constructors;
   try {
     if (typeof (Intl) === 'object' && typeof (Intl.DateTimeFormat) === 'function' && typeof (Intl.NumberFormat) === 'function') {
-      constructors = [
-        Intl.DateTimeFormat,
-        Intl.NumberFormat,
-      ];
+      /**
+       * intl-locales-supported accesses the 'supportedLocalesOf' property of each of these constructors.
+       * When certain polyfills are used, the polyfill may not have the 'supportedLocalesOf' property.
+       * For example, when using the date-time-format-timezone Intl.DateTimeFormat becomes Intl.DateTimeFormatPolyfill which does not support this property.
```

Comment:
Could we reference the logged issue here?

Reply:
Added github issue reference: https://github.com/cerner/terra-core/pull/2915/commits/4aaff5e3c5158f50ced1cf662dfe2da58dd31508